backend/app/api/endpoints/dataset_items.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from datetime import datetime
+from typing import Annotated
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, File, HTTPException, Query, UploadFile, status
+from starlette.responses import StreamingResponse
+
+from app.api.dependencies import get_dataset_item_id, get_dataset_item_service
+from app.schemas import DatasetItem, DatasetItemsWithPagination
+from app.schemas.base import Pagination
+from app.services import DatasetItemService, ResourceNotFoundError
+
+router = APIRouter(prefix="/api/dataset/items", tags=["Dataset items"])
```

Comment:
```suggestion
router = APIRouter(prefix="/api/projects/{project_id}/dataset/items", tags=["Datasets"])
```
The prefix should include the project